### ISSUE TITLE
[place] climate charts should use per capita for comparisons

### ISSUE THREAD

**Issue by beets** [datacommonsorg/website#1281]
for sv's like emissions and water withdrawal rate, the comparisons charts should use per capita so they can be meaningful with containing places.

![image](https://user-images.githubusercontent.com/6052978/140013576-00a5ccd7-ab5e-4bc9-a87c-d282a0f470f2.png)

**Comment by shifucun** [comment]
I talked with @pradh during PR review, he suggested the water withdrawal rate is not quite relevant to population, as there are inter places water transfer. I wonder if there are other good normalized metric

**Comment by pradh** [comment]
Right, I thought per capita could be odd here because this is not water consumption, but water withdrawal from source in a state (for various uses not limited to that state).

**Comment by beets** [comment]
in this case, maybe we can drop the contained-in comparisons? we don't have an option for that yet in the chart config

we could do it for emissions though:
![image](https://user-images.githubusercontent.com/6052978/140105269-f7d469ab-2395-4c76-94c5-3e8a2d0e7505.png)

slightly related: can we move the disasters charts to the bottom of the climate page? the transition from emissions -> disasters -> cancer -> emissions is a bit jarring

**Comment by shifucun** [comment]
We can disable the related chart in config, shall we do it for this chart?. Right now the charts are ordered by titles, with the increasing number of charts, we could introduce sub-group concept in chart config, and only order by title in each sub group?

**Comment by beets** [comment]
yea, lets disable related chart for now..

also, not sure how i missed the ordering.  +1 to subgroups for topic pages too.
